src/providers/uploadsManager/useStartFileUpload.tsx

```diff
@@ -29,6 +30,39 @@ export const useStartFileUpload = () => {
   const addAsset = useUploadsStore((state) => state.addAsset)
   const setUploadStatus = useUploadsStore((state) => state.setUploadStatus)
   const assetsFiles = useUploadsStore((state) => state.assetsFiles)
+  const uploadStatus = useUploadsStore((state) => state.uploadsStatus)
+  const {
+    getDataObjectAvailability,
+    stopPolling,
+    startPolling,
+    dataObjectAvailability,
+    variables,
+  } = useDataObjectAvailabilityLazy({
+    fetchPolicy: 'network-only',
+  })
+
+  useEffect(() => {
+    if (
+      !variables?.joystreamContentIdEq ||
+      uploadStatus[variables?.joystreamContentIdEq]?.lastStatus !== 'proccessing'
+    ) {
+      return
+    }
+    if (dataObjectAvailability === 'PENDING') {
+      startPolling?.(3000)
```

Comment:
let's move this to `config` and let's call it something like `ASSET_POLLING_INTERVAL` and make this value 2000 for now